### DISCUSSION TITLE
Noise box in mission brief

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#205] [General]
Just found an implementation of filling a box with noise:

![synII184](https://github.com/user-attachments/assets/34d3df12-756d-4040-9a5a-50a60d932113)

The noise changes between frames, like "the color of television tuned to a dead channel" (cite from "Neuromancer").

Do we have any idea of how to use this feature?

**Comment by Moburma** [comment]
Ahh, I've seen that before:
https://tcrf.net/File:SynWars-SYNII008.GIF
Now, what to actually use it for, that's a different question....

**Comment by Moburma** [comment]
I guess a more serious reply, can it be made as a Netscan entry effect? I guess then it can be used to make the map become fuzzy on that entry. Not sure how good that would be, but it could be useful for "hackerman"  effects in e.g. the unguided campaign (in my mind it would often be Vissick breaking into public systems to talk to people en masse, so you could tie that in there).

**Reply by mefistotelis** [reply]
Currently it's just noise, but yeah maybe a better effect would be to mix the noise with existing map - making the map unstable / fuzzy and noisy.

I actually think the final level, or final 2 levels, should have such effect and snow on the level, to emphasize someone is messing with the weather control system.
